new.html

```diff
@@ -0,0 +1,586 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>New Evaluation-MockUp</title>
+	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="css/new.css">
+	<link rel="stylesheet" type="text/css" href="css/common.css">
+</head>
+<body>
+	<!-- Header --> 
+	<header class="header">
+		<nav class="nav">
+			<div id="logo" class="logo "></div>
+			<div id="navigation" class="navigationClass">
+				<a href="evaluations.html">
+					<div id="button1" class="button button1 formatted">
+					Evaluations
+					</div>
+				</a>
+				<div id="button2" class="button button2 formatted selected">
+					New Evaluation
+				</div>
+		</div>
+		<span class="headerSpan formatted"><a class="logOutLink" href="login.html">Logout</a></span>
+	</nav>
+</header>
+
+<!-- Main section --> 
+<section>
+	<div class="main-section">
+		<form action="" method="">
+			<div class="form-section">
+				<div class="form-section-first">
+					<input type="text" placeholder="Candidate">
+					<input type="text" placeholder="Interviewer">
+					<input type="date" class="inputDate">
+
+				</div>
+				<span class="formatted blue">Tehnical level</span></br>
+				<table>
+					<tr class="firstRow">
```

Comment:
There are a couple of other `table`-related elements that you can use to further separate your markup for tabular data: `thead` / `tbody` / `tfoot`. Including these might make the need for an extra class here simply go away, since you'd know that your table header should always be included in your `thead`, so you could build your selectors to leverage that knowledge.